examples/fetch-file-from-kubo/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "fetch-file-from-kubo",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview"
+  },
+  "overrides": {
+    "libp2p": "../../../js-libp2p"
+  },
+  "dependencies": {
+    "@chainsafe/libp2p-noise": "^9.0.0",
+    "@chainsafe/libp2p-yamux": "^2.0.0",
+    "@multiformats/multiaddr": "^11.0.3",
+    "blockstore-core": "^2.0.1",
+    "ipfs-bitswap": "../../../../ipfs/js-ipfs-bitswap/",
+    "js-libp2p-webtransport": "../../",
+    "libp2p": "^0.40.0"
+  },
```

Comment:
```suggestion
  "scripts": {
    "start": "vite",
    "build": "tsc && vite build",
    "preview": "vite preview"
  },
  "dependencies": {
    "@chainsafe/libp2p-noise": "^9.0.0",
    "@libp2p/webtransport": "../../",
    "@multiformats/multiaddr": "^11.0.3",
    "blockstore-core": "^2.0.1",
    "ipfs-bitswap": "^13.0.0",
    "libp2p": "^0.40.0"
  },
```

Reply:
Creating a `"start"` script lets the user run `npm start` instead of `npm run dev`.